### PR TITLE
Add average speed calculation to measurements

### PR DESCRIPTION
**Pull request by muthenberg**
Please ignore all the stuff below Pods.

### CODE REVIEW

File: DataCapturing/Source/Model/Measurement.swift [hunk 1]
```diff
@@ -33,29 +33,11 @@ Instances of this class represent a single data capturing session, framed by cal
...
-    /// A flag, marking this `Measurement` as either synchronized or not.
-    public var synchronized: Bool
-    /// The UNIX timestamp in milliseconds since the 1st of January 1970, when this measurement was started.
-    public let timestamp: Int64
-    /// The calculated length of the `Measurement`. See `DistanceCalculationStrategy` for further details.
-    public var trackLength: Double
-    /// The user events that occurred during this `Measurement`.
-    public var events: [Event]
-    /// The tracks containing all the data this `Measurement` has captured.
-    public var tracks: [Track]
```
Comment (muthenberg) [inline DataCapturing/Source/Model/Measurement.swift:57]: These have just been moved below.

File: DataCapturing/Source/Model/Measurement.swift [hunk 2]
```diff
@@ -122,6 +104,58 @@ public class Measurement: Hashable, Equatable {
...
+                counter += 1
+            }
+        }
+
+        if counter==0 {
+            return 0.0
+        } else {
+            return sum/Double(counter)
+        }
+    }
```
Comment (muthenberg) [inline DataCapturing/Source/Model/Measurement.swift:155]: This is the only changed part. Everything else has only been moved around.

Comment (hb0) [reply DataCapturing/Source/Model/Measurement.swift:155]: The only question (in comparision with the Android code) would be:

Didn't we want to use the `LocationCleaningStrategy` in average speed calculation to not count the standing time "speed ~ 0"?

See here: https://github.com/cyface-de/android-backend/pull/257#discussion_r1068095043

**Review by hb0 — CHANGES_REQUESTED** [review]
See comment

**Review by hb0 — APPROVED** [review]

**Pull request merged**